lib/suites/Ed25519Signature2018.js

```diff
@@ -40,7 +40,11 @@ module.exports = class Ed25519Signature2018 extends LinkedDataSignature {
       JSON.stringify(header), {forge});
 
     let encodedSignature;
-    if(this.injector.env.nodejs) {
+    if(options.signer) {
+      // a custom signing API has been provided
+      const signingString = _createSigningString();
+      encodedSignature = await options.signer.api({signingString});
```

Comment:
Should this be `signer.sign()`? (instead of `signer.api()`?)

`.api()` reads like it returns a whole API, whereas I assume it's an invocation to sign something?

Reply:
I wonder if `signer.api.sign` would not be the thing?

Reply:
`signer.api.sign()` seems reasonable; my only question then is - are there any other functions in `.api`, other than sign?